standard.tex

```diff
@@ -156,21 +153,18 @@ \section{Documentation Standards}
 \item Any circuit you build should be thoroughly documented in a schematic. Schematics should be correct, clear, and well-organized. The primary purpose of a schematic is to communicate the circuit to someone who hasn't seen it before. A messy schematic will make your circuit harder to review.
 \item Some guidelines can be found at \url{https://electronics.stackexchange.com/questions/28251/rules-and-guidelines-for-drawing-good-schematics}.
 \item The components in your schematics should match what you intend to use in the circuit. If you are using a particular component, label it as such (and provide a datasheet - see above). This will help identify potential issues in design reviews.
-\item If your schematic editor doesn't have a symbol for the part you intend to use, make a custom one (all decent editors support this). Eg. a TL780 voltage regulator should not be represented by a BD244 transistor.
-\item Provide both the PDFs and original files for schematics and PCBs.
+\item Provide both the PDFs and original files for final schematics and PCBs.
 \end{itemize}
 \end{enumerate}
-%end of documentation section -- }}}
+%-- }}}
 
-%start of design guidelines section -- {{{
+%Design Reviews -- {{{
 \section{Design Reviews}
-% Expand this into "Design Guidelines" as more stuff accumulates.
 If you're designing any part of a system, you should be conducting design reviews. This applies whether there are multiple people working on the system or if you're the only one. If you're alone on a project, this is a good opportunity to make someone else familiar with your work. Apart from resulting in a better circuit, design reviews will also help ensure that you are never the only one who knows how your project should work.
 \begin{itemize}
-\item  Conduct design reviews on every major revision of your system. This means bringing in another person to discuss your circuit. Keep a record of what was discussed and decided in the review. When presenting a new revision, indicate what has been changed since the last review.
-\item Design reviews should be thorough. Even innocent looking sections should be discussed. Chances are you won't find a bug there, but it will save you pain down the line. Discuss failure modes and alternative solutions.
-\item The person doing your review should know what they're doing.
-\item If appropriate, invite junior members to listen in on the review. They will benefit from being part of the review process and will become better designers and reviewers in the future.
+\item  Conduct design reviews on every major revision of your system. This means bringing in another person to discuss your circuit. Keep a record of what was discussed and decided in the review. When presenting a new revision, indicate what has been changed since the last review (a commit message is enough).
```

Comment:
Every major revision is ambiguous

I'd say it at least needs to happen:
- After you have a schematic before you start designing a PCB
- After you have a PCB mostly done (except minor things like silkscreen and such)
- Immediately before you tape out gerbers. You need a commit that has gerbers in it that has been reviewed (in gerb view, don't review gerbers from a kicad_pcb file), and passed, and then those gerbers (which are SHA verified) are what go to the fab

Reply:
Fixed.